generator/idl_finder.js

```diff
@@ -1,8 +1,24 @@
+var fs = require('fs');
+var path = require('path');
+
+async function getNidlFiles(dir, files) {
+  files = files || [];
+  var list = fs.readdirSync(dir);
+  list.forEach(function(file) {
+    file = path.resolve(dir, file);
+    if (fs.statSync(file).isDirectory() && !files === 'node_modules'){
+      getNidlFiles(file, files);
+    } else if( file.includes('.nidl')) {
+      files.push(file);
+    }
+  });
+  return files;
+}
+
 async function findAll() {
-  // TODO(zino): Should implement this function.
-  return [
-    `${__dirname}/../examples/hello.nidl`
-  ];
+  var packageDir = process.cwd();
+  var nidlList = await getNidlFiles(packageDir);
```

Comment:
IMHO, it's not need to call process.cwd().

Reply:
Okay. I will change it to getNidlFiles(process.cwd()).